internal/wts.go

```diff
@@ -32,6 +32,9 @@ type Gen3FuseConfig struct {
 
 	// An optional parameter the user can provide to retrieve access tokens from Fence
 	ApiKey string
+
+	// An optional parameter the user can provide to talk to WTS from outside the k8s cluster
+	AccessToken string
```

Comment:
-> Can you add info to the README that describes what this parameter does, what use case it matches, and how to populate it? 

-> I think that the name "access token" is generic enough that it could mean anything in this context. Can you rename it to be more specific? For example, "ExternalClusterAccessToken" + "external-cluster-access-token"